spec/system/admin/products_v3/products_spec.rb

```diff
@@ -397,7 +381,6 @@
       end
 
       # Unit popout
-      # TODO: prevent empty value
```

Comment:
Sorry I didn't see this notification earlier. Yes thanks this can be removed, it should have been removed in https://github.com/openfoodfoundation/openfoodnetwork/commit/cf31d09ad8ac60626db7aecbda78e93841bbc6ea

Reply:
Thanks and sorry @dacook , this was bad commit history on my end. I should have squashed these two commits before.